app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -18,12 +18,19 @@
             GOV.UK
           </span>
         </span>
+
         <% if product_name %>
         <span class="govuk-header__product-name">
           <%= product_name %>
         </span>
         <% end %>
       </a>
+
+      <% unless environment == "production" %>
```

Comment:
Can we move this inside the anchor so Assistive Tech users know what the environment is when they get to the landmark?